File: Upcoder/Lập trình cơ bản/7.Other/Tính tiền/LTCB_GK.Cau2.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    double S, E, T;
    double gia = 30000;
    double gia2 = 30000 * 0.7;
    while (cin >> S)
    {
        cin >> E;
        T = E - S;
        if(T <= 3)
        {
            T = T * 30000;
            cout << T << endl;
        }
        else
        {
            T = 3 * 30000 + (T - 3) * (30000 * 0.7);
            cout << T << endl;
        }
    }
    return 0;
}

File: Upcoder/Lập trình cơ bản/7.Other/Tính tiền/LTCB_GK.Cau2.py
while True:
    try:
        S, E = map( float, input().split()) 
        T = E - S           
        gia = 30000
        gia2 = 30000 * 0.7  
        if T <= 3:
            T = T * gia
            print(int(T))
        else:
            T = 3 * gia + (T - 3) * gia2
            print(int(T))
    except EOFError:
        break
